Dockerfile

```diff
@@ -1,6 +1,27 @@
-FROM alpine:3.7
-MAINTAINER Nic Cope <n+docker@rk0n.org>
+FROM node as node
```

Comment:
I'd like to pin this too. Maybe to `9.8-alpine`?

Reply:
Sure, I didn't know which version to pin to, so great that you point it out, i'll include that in the next commit.